src/lighteval/models/vllm/vllm_model.py

```diff
@@ -191,7 +194,7 @@ def _create_auto_tokenizer(self, config: VLLMModelConfig, env_config: EnvConfig)
             config.pretrained,
             tokenizer_mode="auto",
             trust_remote_code=config.trust_remote_code,
-            tokenizer_revision=config.revision,
+            revision=config.revision + (f"/{config.subfolder}" if config.subfolder is not None else ""),
```

Comment:
Important fix: the argument is indeed `revision`
https://github.com/vllm-project/vllm/blob/04139ade599eedd493ce8effcda7ceabb57f2fb5/vllm/transformers_utils/tokenizer.py#L92

Reply:
Thanks for the update re-revision - not sure about why you're doing the next addition to the path howver?